extensions/windows-event-log/TailEventLog.h

```diff
@@ -120,10 +121,10 @@ class TailEventLog : public core::Processor {
       (LPTSTR)&lpMsg,
       0, NULL);
 
-    logger_->log_debug("Error %d: %s\n", (int)error_id, (char *)lpMsg);
+    logger_->log_debug("Error %d: %s\n", static_cast<int>(error_id), reinterpret_cast<char *>(lpMsg));
```

Comment:
There's a memory leak here. `lpMsg` should be freed with `LocalFree` according to [MSDN](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-formatmessage#FORMAT_MESSAGE_ALLOCATE_BUFFER).

Reply:
thanks, fixed in [95aeacd](https://github.com/martinzink/nifi-minifi-cpp/commit/95aeacd78262361de87609058f1f208b01e4fe3b)